src/utils/utils/utils.ts

```diff
@@ -1,5 +1,13 @@
+import { uniqueNamesGenerator, adjectives, animals } from 'unique-names-generator';
+
 export const networkConsole = console;
 
 export const isEmpty = (obj) =>
-  [Array, Object].includes((obj || {}).constructor) &&
-  !Object.entries(obj || {}).length;
+  [Array, Object].includes((obj || {}).constructor) && !Object.entries(obj || {}).length;
+
+export const generateNADName = (prefix: string): string => {
```

Comment:
It's better to avoid a specific name when moving to a more shared location so we can use it in different places. Please change to generateName